ui/components/app/permission-page-container/permission-page-container.component.js

```diff
@@ -145,22 +145,26 @@ export default class PermissionPageContainer extends Component {
       approvePermissionsRequest,
       rejectPermissionsRequest,
       selectedAccounts,
+      requestedChainIds,
     } = this.props;
 
     const approvedAccounts = selectedAccounts.map(
       (selectedAccount) => selectedAccount.address,
     );
 
-    const requestedSessionsScopes = getRequestedSessionScopes(
-      _request.permission,
+    const requestedCaip25CaveatValue = getRequestedCaip25CaveatValue(
+      _request.permissions,
     );
-    const approvedChainIds = getPermittedEthChainIds(requestedSessionsScopes);
```

Comment:
this component was previously ignoring any requested chainIds, but i suppose it doesn't really matter since this flow only works for snaps which currently can't request chainIds in the first place. Just pointing out why this change was made

Reply:
I have attempted to extract this to a separate PR, since it seems unrelated to the main purpose of this PR: https://github.com/MetaMask/metamask-extension/pull/30443